docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -369,6 +369,7 @@ statements, the connector supports the following features:
 * :doc:`/sql/alter-table`
 * :doc:`/sql/create-schema`
 * :doc:`/sql/drop-schema`
+* :doc:`/sql/comment` only supports ``COMMENT ON TABLE``
```

Comment:
The convention until now has been to add a heading called `COMMENT` and mention what is not supported.

See ALTER TABLE just below for example.

(I don't have a strong preference but would like consistency).

Reply:
> mention what is not supported.

This is fragile for future changes. We have to visit all connectors' page when adding a new option to the statement.

Reply:
That is a very good point and good justification for only mentioning what is supported.

cc: @jhlodin Maybe you'd want to update existing docs. It's very easy for docs to go out of date since when adding features it's not obvious that I need to search if it has been excluded somewhere. Much easier to explicitly mention what is supported.

Reply:
I agree with the philosophy of saying what's supported rather than unsupported, we can look into standardizing that. From a maintenance perspective, isn't it still the same work to find applicable connectors and append the new feature to those that now support it? Unless the connector supports *all* uses of a given statement, we'll still need to explicitly list which ones it does (whether that's framed as inclusive or exclusive).

Reply:
I'm still happy to standardize that, I'm just not sure how that makes this part of the process easier.